giskard/client/dtos.py

```diff
@@ -28,6 +35,66 @@ class TestSuiteDTO(ConfiguredBaseModel):
     function_inputs: List[TestInputDTO]
 
 
+class TestSuiteExecutionResult(str, Enum):
+    IN_PROGRESS = ("IN_PROGRESS",)
```

Comment:
Why ( "<str>", ), and "ERROR" ? are the parenthesis needed ?

Reply:
My bad I added unnecessary  commas and the pre-commit reformatted like that